operational/aws/long_running_instances/long_running_instances.pt

```diff
@@ -246,9 +253,9 @@ escalation "email" do
 end
 
 escalation "esc_stop_or_terminate" do
-  automatic false
+  automatic contains($param_automatic_action, "Stop/Terminate Instances")
```

Comment:
there should be two actions, stop or terminate

Reply:
Done!